develop_requirements.txt

```diff
@@ -0,0 +1,22 @@
+# sources
+-e https://github.com/morepath/reg.git#egg=reg
```

Comment:
I'm curious to see what the plan is here, I saw the thing was a work in progress. The goals are:

* us, the core developers with write access to the Morepath repos, can get writeable checkouts.

* a new developer that does not have write access can get a checkout at all. If they have to upload an ssh key to github to get it, so be it, but they should be able to do it without write access. This needs to be tested if we haven't already..

I can see why having two requirements files is problematic as that would require a duplication of the source URLs, something we want to avoid. If this is impossible without two requirements files we need to discuss our options.